synapse/config/server.py

```diff
@@ -649,6 +651,39 @@ class LimitRemoteRoomsConfig:
                 )
             )
 
+        manhole_settings = config.get("manhole_settings") or {}
+        manhole_username = manhole_settings.get("username") or "matrix"
+        manhole_password = manhole_settings.get("password") or "rabbithole"
```

Comment:
I was using the `get() or None` way in case the config file had `username: ` (i.e. had set no username) in which case you would get an error when trying to setup the manhole (similarly this sets "rabbithole" as the password if they try and set no password)

Reply:
oh right. I don't think that's a thing we should support - users should either set it to a string, or not set it at all.